### PR TITLE
WEBEU2 - WebDB-I-Challenge - Isaac Aderogba 🇮🇪 

### PR DESCRIPTION
**Pull request by IsaacAderogba**

### CODE REVIEW

File: queries.md [hunk 1]
```diff
@@ -2,16 +2,34 @@
...
+VALUES ("The Shire", "Bilbo Baggins", "1 Hobbit-Hole", "Bag End", 111, "Middle Earth")
+
 ## Update Bilbo Baggins record so that the postal code changes to "11122"
 
+[x] UPDATE Customers SET PostalCode=11122 WHERE ContactName="Bilbo Baggins";
+
 ## (Stretch) Find a query to discover how many different cities are stored in the Customers table. Repeats should not be double counted
 
+[x] SELECT Count(*) FROM Customers GROUP BY City;
+
```
Comment (delbaoliveira) [inline queries.md:32]: Nice to see you've found a different way of doing this!

File: routes/accountsRouter.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+const router = express.Router();
+
+router.get("/", async (req, res, next) => {
+  try {
+    const {limit, sortby, sortdir} = req.query;
+    const accounts = await Accounts.get(limit, sortby, sortdir);
+    res.status(200).json(accounts);
+  } catch (err) {
+    next(err);
```
Comment (delbaoliveira) [inline routes/accountsRouter.js:12]: Could you explain why you are calling `next()` inside an endpoint if it's not a middleware? Would love to hear your reasoning behind this. :)

**Review by delbaoliveira — COMMENTED** [review]
Good job Isaac! You're doing great. I've left some comments below 👇

File: routes/accountsRouter.js (hunk 2 above)
Comment (IsaacAderogba) [reply routes/accountsRouter.js:12]: Sure, `next()` in this case passes the caught error along to my error-handling middleware. Perfectly fine to call next here!

Comment (delbaoliveira) [reply routes/accountsRouter.js:12]: Thanks! That makes sense!

**Pull request merged**